go/private/go_toolchain.bzl

```diff
@@ -33,6 +34,7 @@ def _go_toolchain_impl(ctx):
           "TMP": tmp,
       },
       actions = struct(
+          env = action_with_go_env,
```

Comment:
Will this be public? Should be documented in toolchains.rst.

Reply:
It will be, but I don't want people using it just yet, so I will document once it's all in (there is a lot more changes to come on the way to cross compiling standard libraries on demand)